labelbox/schema/es_filters.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from datetime import datetime, timezone
+from typing import Collection, Dict, Tuple, List, Optional
+from labelbox.typing_imports import Literal
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+SEARCH_LIMIT_PER_EXPORT_V2 = 2_000
+ISO_8061_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
+
+
+def build_id_filters(
```

Comment:
I have refactored this because I thought it would be useful for my story. As we have changed the direction from using the `DeleteDataRowsByQuery` mutation to `DeleteDataRows`, we do not need to use this code. Still I have decided to keep the refactor since it is good and has prior unit tests.